maraboupy/MarabouNetworkNNet.py

```diff
@@ -13,40 +13,180 @@
-                      with the normalization already incorporated.
-    Attributes:
-        numLayers (int): The number of layers in the network
-        layerSizes (list of int): Layer sizes
-        inputSize (int): Size of the input
-        outputSize (int): Size of the output
-        maxLayersize (int): Size of largest layer
-        inputMinimums (list of float): Minimum value for each input
-        inputMaximums (list of float): Maximum value for each input
-        inputMeans (list of floats): Mean value for each input
-        inputRanges (list of floats): Range for each input
-        outputMean (float): Mean value of outputs
-        outputRange (float): Range of output values
-        weights (list): Network weight matrices, where the outer index corresponds to layer number
-        biases (list): Network bias vectors, where the outer index corresponds to layer number
-     """
-    def __init__ (self, filename, normalize=False):
+    """ Abstract class representing a Marabou network from an NNet file
+        Inherits from MarabouNetwork class
+
+        Attributes:
+
+        initialized in __init__:
+            numLayers (int): The number of layers in the network
+            layerSizes (list of int): Layer sizes
+            inputSize (int): Size of the input
+            outputSize (int): Size of the output
+            maxLayersize (int): Size of largest layer
+            inputMinimums (list of float): Minimum value for each input
+            inputMaximums (list of float): Maximum value for each input
+            inputMeans (list of floats): Mean value for each input
+            inputRanges (list of floats): Range for each input
+            outputMean (float): Mean value of outputs
+            outputRange (float): Range of output values
+            weights (list): Network weight matrices, where the outer index corresponds to layer number
+            biases (list): Network bias vectors, where the outer index corresponds to layer number
+
+        initialized elsewhere:
+            b_variables (list): List of b variables
+            f_variables (list): list of f variables
+
+        attributes from parent MarabouNetwork:
+            numVars (int): Total number of variables to represent network
+            equList (list of :class:`~maraboupy.MarabouUtils.Equation`): Network equations
+            reluList (list of tuples): List of relu constraint tuples
+            maxList (list of tuples): List of max constraint tuples
+            varsParticipatingInConstraints (set of int): Variables involved in some constraint
+            lowerBounds (Dict[int, float]): Lower bounds of variables
+            upperBounds (Dict[int, float]): Upper bounds of variables
+            inputVars (list of numpy arrays): Input variables
+            outputVars (numpy array): Output variables
+
+    """
+
+
+    def __init__(self, filename='', normalize=False):
+        """Constructs a MarabouNetworkNNet object from an .nnet file
```

Comment:
No docstring needed for __init__, since it is already covered by the class docstring.